app/views/local_transaction/search.html.erb

```diff
@@ -13,9 +13,9 @@
   <% if @publication.introduction.present? %>
     <section class="intro">
       <div class="get-started-intro">
-        <%= render "govuk_publishing_components/components/govspeak", {
-          content: @publication.introduction.html_safe
-        } %>
+        <%= render "govuk_publishing_components/components/govspeak", {} do %>
+          <%= raw @publication.introduction.html_safe %>
```

Comment:
is this effectively doing .html_safe twice at the moment? I think you can drop it inside the raw

Reply:
Fixed